s/frontend/components/modals/views/delete-thread/delete-thread-modal-view.ts

```diff
@@ -0,0 +1,37 @@
+
+import {html} from "lit"
+import {ModalControls} from "../../modal-types.js"
+import {NestedComment} from "../../../../models/commenting/commenting-types.js"
+
+export async function deleteThreadModalView({
```

Comment:
this modal isn't just for deleting threads, so it shouldn't be called "delete *thread* modal view"

maybe something more general, like "delete *post* modal view"